prusti-viper/src/encoder/procedure_encoder.rs

```diff
@@ -2569,66 +2594,55 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
         self.slice_created_at.insert(location, encoded_lhs);
 
         // TODO: what do we actually do here? this seems a littly hacky.
```

Comment:
Unsure the comment is still relevant. Is it just that we rely on "sdtlib internals"? If so, let's say that in the comment.

Reply:
See below